lib/miq_automation_engine/service_models/miq_ae_service_user.rb

```diff
@@ -3,6 +3,7 @@ class MiqAeServiceUser < MiqAeServiceModelBase
     expose :current_group,  :association => true
     expose :current_tenant, :association => true
     expose :vms,            :association => true
+    expose :miq_groups,     :association => true
```

Comment:
Yes; however as long as it goes through MiqGroup#get_filters (which is delegated to the entitlement) I think we're fine. We'll probably deprecate the MiqGroup method in the future.